src/components/App/App.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { Route } from 'react-router-dom';
+import Header from '../Header/Header';
+import PaletteForm from '../PaletteForm/PaletteForm';
+import './App.scss';
+
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      projects: []
+    }
+  }
+
+  // componentDidMount() {
+
+  // }
+
+  render() {
+
+    return (
+      <div className='App'>
+        <Header />
+        <main>
+          <Route exact path='/' render={() => <PaletteForm />}/>
```

Comment:
We will need to add the projects path included in the apps render method so we can pass the projects as props.